protos/dcs/common/v0/common.proto

```diff
@@ -264,12 +264,28 @@ message Weapon {
  * other game behaviour aside from being destroyable
  */
 message Static {
+  // The DCS generated ID
+  uint32 id = 1;
+  // The DCS type-name of the static
+  string type = 2;
+  // The name of the static
+  string name = 3;
+  // The coalition the static belongs to
+  Coalition coalition = 4;
+  // The position of the static
+  Position position = 5;
```

Comment:
A minor note - and we can evolve this in a later PR - cargo would have additional properties such as weight.

If statics are being transported, or is attached to a moving object (e.g. statics on CV), statics would have a velocity.